errbot/backends/telegram_messenger.py

```diff
@@ -191,9 +191,14 @@ def __init__(self, config):
         self.telegram = None  # Will be initialized in serve_once
         self.bot_instance = None  # Will be set in serve_once
 
-        compact = config.COMPACT_OUTPUT if hasattr(config, 'COMPACT_OUTPUT') else False
-        enable_format('text', TEXT_CHRS, borders=not compact)
-        self.md_converter = text()
+        self.default_send_options = dict(parse_mode='Markdown')
```

Comment:
I would like to see the default parse mode remain text in order to remain consistent with the current behavior (and because the markdown parsing mode Telegram has is a little iffy.. I deliberately used the text converter because it typically produced better results).